flame_utils/core/model.py

```diff
@@ -407,8 +408,11 @@ def run(self, bmstate=None, from_element=None, to_element=None, monitor=None):
             element.
         monitor : list[int] or list[str] or 'all'
             List of element indice or names selected as states monitors, if
-            set -1, will be a list of only last element. if set 'all',
-            will be a list of all elements.
+            set -1, it returns the finite beam state. if set 'all', it returns
```

Comment:
"the finite beam state" should be "a list of only last element"?

Reply:
Do we need a new test for `include_initial_state` argument?

Reply:
1. I think I can roll back to the original comment.
2. I will add new test case.